lexer/src/lexer.rs

```diff
@@ -27,7 +27,6 @@ macro_rules! match_op {
     ($input:expr, $span:ident, $len:expr, $right:expr) => {{
         // Unwraps are safe, because this is only called after token of length $len,
         // is already matched and thus contained in $input
-        debug_assert!($len >= 1);
```

Comment:
Maybe wait on rust-lang/rust-clippy#3740 or allow the clippy lint here instead of removing it.